pages/common/ed.md

```diff
@@ -1,36 +1,33 @@
 # ed
 
 > The original Unix text editor.
+> See also: `awk`, `sed`.
 > More information: <https://www.gnu.org/software/ed/manual/ed_manual.html>.
 
-- Start ed, editing an empty document (which can be saved as a new file in the current directory):
+- Start an interactive editor session with an empty document:
 
 `ed`
 
-- Start ed, editing an empty document, with `:` as a command prompt indicator:
+- Start an interactive editor session with an empty document and the specified prompt:
 
-`ed -p :`
+`ed --prompt="> "`
 
-- Start ed editing an existing file (this shows the byte count of the loaded file):
+- Start an interactive editor session with user-friendly errors:
```

Comment:
The standard errors are hard to understand?

Reply:
According to `info ed` standard errors are very concise and may be hard to understand:

> Toggles the printing of error explanations. By default,
     explanations are not printed. It is recommended that ed scripts
     begin with this command to aid in debugging.

You can find this note in `H` command description (it is equivalent to `--verbose` option).